src/context/Language.js

```diff
@@ -5,3 +5,16 @@ import * as React from 'react';
 const defaultLanguage = 'en';
 
 export const LanguageContext = React.createContext(defaultLanguage);
+
+export const withDefaultLanguage = <Props>(
+  Component: React.ComponentType<{ defaultLanguage: string } & Props>,
+) =>
+  function withLanguageHOC(props: Props) {
+    return (
+      <LanguageContext.Consumer>
+        {(defaultLanguage: string) => (
```

Comment:
`defaultLanguage` is rather unfortunate name for language prop

Reply:
:smile: I don't find it that crazy offtopic, but you've got a point. What about `language`? :book: